syft/formats/common/spdxhelpers/to_format_model_test.go

```diff
@@ -169,93 +168,6 @@ func Test_toFileChecksums(t *testing.T) {
 	}
 }
 
-func Test_fileIDsForPackage(t *testing.T) {
```

Comment:
The `hasFiles` field, which was using this function is not documented in the SPDX spec but is in the RDF and JSON schemas. However, it has been deprecated since 2.0 in favor of using the `CONTAINS` relationship, which we are already including. See: https://spdx.org/rdf/terms/#d4e4761

Reply:
is there an equivalent functionality that needs to tested now that this test is being removed?